input/pagecontent/StructureDefinition-au-erequesting-imaging-request-intro.md

```diff
@@ -0,0 +1,9 @@
+### Usage scenarios
+The following are supported usage scenarios for this profile:
+
+- Query for a request for a diagnostic imaging service for a patient
+- Record or update a request for a diagnostic imaging service for a patient
+- 
+
```

Comment:
With the coding introduced a STU Note must be added explaining that the RANZCR binding is coming.

Reply:
Added STU Note